optuna/integration/wandb.py

```diff
@@ -52,11 +55,11 @@ def objective(trial):
                 return (x - 2) ** 2
 
 
+            study = optuna.create_study(multirun_tag="my_study")
```

Comment:
I'm not sure but it might be as follows:
```suggestion
            study = optuna.create_study(study_name="my_study")
```

Reply:
Oops, forgot to fix that. Working on it.